qdrant-landing/content/documentation/data-ingestion-beginners.md

```diff
@@ -298,33 +239,34 @@ if __name__ == "__main__":
         text_embedding, image_embedding, points, text_review, product_image = [], [], [], "", ""
```

Comment:
we don't need `text_embedding`, `image_embedding anymore` since we're computing them via fastembed